modules/sbo-unbinding-workloads-from-a-backing-service.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * /applications/connecting_applications_to_services/binding-workloads-using-sbo.adoc
+
+[id="sbo-unbinding-workloads-from-a-backing-service_{context}"]
+= Unbinding workloads from a backing service
+
+You can unbind a workload from a backing service by using the `oc` tool.
+
+. To unbind a workload from a backing service, delete the `ServiceBinding` custom resource (CR) linked to it :
+
+[source,terminal]
+----
+$ oc delete ServiceBinding <.metadata.name>
```

Comment:
```suggestion
$ oc delete ServiceBinding <name_of_ServiceBinding_CR>
```
See the SSG section on "User-replaced values" for guidance on formatting. Also, if you change `.metadata.name` to `name_of_ServiceBinding_CR` then you can delete the whole `where` section below (as I have done)

Reply:
Nope, I would like to retain this because ".meatadata.name" is the attribute in the CR of the service binding resource.

Our OCP guideline has the provision for such rare usage needs. Refer [here](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output): "If you must provide additional information on what a line of a code block represents and the use of callouts is impractical, you can use a description list to provide information about the variables in the code block. Using callouts might be impractical if a code block contains too many conditional statements to easily use numbered callouts or if the same note applies to multiple lines of the codeblock."

I am retaining the content as is. Apologies for not being able to incorporate your suggestions. Thanks for your understanding and cooperation :)

Reply:
@Srivaralakshmi Thanks for the clarification. I learned something new!